packages/devtools_app/lib/src/theme.dart

```diff
@@ -339,6 +339,12 @@ const wideSearchTextWidth = 400.0;
 const defaultSearchTextWidth = 200.0;
 const defaultTextFieldHeight = 36.0;
 
+/// Default color of cursor and color used by search's TextField.
+Color searchColor = Colors.lightBlueAccent[400];
```

Comment:
Comment on where this is from. Does this match the default on a specific platform? Comment why we don't just use the default so someone viewing the theme understands why it makes sense to customize this to improve our testability and consistency.

Reply:
Added comment on why we need a consistent color for golden snapshot compares.

After looking carefully at the theme-ing code the default is Color.blue (blue[500]) checked with a color picker to confirm.